### ISSUE TITLE
Inaccessible Fake buttons

### ISSUE THREAD

**Issue by jpdevries** [Thinkful/ui#359]
> Not sure if this is directly related to the UI library, as we don't seem to have a component for that

### User Story 
As a keyboard user, I should be able to focus, access, and use the "buttons" on the page. As a screen reader user, I should find the buttons in my rotor. 

### Explanation

This is an example of a WCAG violation that automated testing will most likely not catch.

`<div class="button">` should never be used for an interactive element. Ideally the native `<button>` element should be used as it is implicitly accessible. If for some odd reason `<button>` cannot be used, use `<div role="button">` and use a `div[role="button"]` selector rather than `div.button`.

![](http://j4p.us/1A0P1U42262a/Screen%20Shot%202017-03-30%20at%204.02.27%20PM.png)

**Comment by jpdevries** [comment]
Once we get the buttons themselves implicitly or declaratively accessible the next step would be to make them more descriptive. Assuming they were in the accessibility tree, the would all read as "RSVP". That isn't very helpful, as it leaves a screen reader user wondering RSVP to what as they cycle through the rotor.

Solutions for this are either use `aria-label` or visually hidden text inside the button so it reads something like "RSVP for FEWD All the things on Mar 31st at 11am".
